cmake/modules/RootCPack.cmake

```diff
@@ -29,12 +29,26 @@ string(REGEX REPLACE "^([0-9]+).*$" "\\1" CXX_MAJOR ${CMAKE_CXX_COMPILER_VERSION
 string(REGEX REPLACE "^([0-9]+)\\.([0-9]+).*$" "\\2" CXX_MINOR ${CMAKE_CXX_COMPILER_VERSION})
 
 #---Resource Files-----------------------------------------------------------------------------------
-configure_file(README.md README.md COPYONLY)
 configure_file(LICENSE LICENSE.txt COPYONLY)
 configure_file(LGPL2_1.txt LGPL2_1.txt COPYONLY)
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_BINARY_DIR}/README.md")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_BINARY_DIR}/LICENSE.txt")
-set(CPACK_RESOURCE_FILE_README "${CMAKE_BINARY_DIR}/README.md")
+if (APPLE)
+  find_program(CONVERTER textutil)
+  if (NOT CONVERTER)
+    message(FATAL_ERROR "textutil executable not found")
+  endif()
+  execute_process(COMMAND ${CONVERTER} -convert html README.md)
+else()
+  find_program(CONVERTER pandoc)
+  if (NOT CONVERTER)
+    message(FATAL_ERROR "pandoc executable not found")
```

Comment:
Won't this make configuration fail if pandoc is not installed, even if a package doesn't need to be generated?

Reply:
I think that's what the PR failures are telling us, indeed ;-)

Reply:
So what's the way forward?

Reply:
I don't think any other package builder needs this; it's just the Mac's. So please drop this `else()` part.

Reply:
But is it okay to have inconsistent behaviour where we convert README.md to README.html in Mac(because CPack cannot handle markdown files on Mac), but leave it as README.md in other OS?

Reply:
That's fine with me, yes.

Reply:
Okay - I removed the else block.